### ISSUE TITLE
Claim: smallNORB - 3D object recognition from shape.

### ISSUE THREAD

**Issue by f2010126** [DagsHub/3D-model-datasets#13]
This issue aims to add the dataset with the following details.

The smallNORB dataset is a datset for 3D object recognition from shape. It contains images of 50 toys belonging to 5 generic categories: four-legged animals, human figures, airplanes, trucks, and cars. The objects were imaged by two cameras under 6 lighting conditions, 9 elevations (30 to 70 degrees every 5 degrees), and 18 azimuths (0 to 340 every 20 degrees). The training set is composed of 5 instances of each category (instances 4, 6, 7, 8 and 9), and the test set of the remaining 5 instances (instances 0, 1, 2, 3, and 5).

Paper with Code Page- [paperswithcode.com/dataset/smallnorb](https://paperswithcode.com/dataset/smallnorb)
Dataset Homepage- [smallNORB-Dataset](https://cs.nyu.edu/~ylclab/data/norb-v1.0-small/)
